acceptance/cluster/localcluster.go

```diff
@@ -436,7 +436,7 @@ func (l *LocalCluster) startNode(node *testNode) {
 		cmd = append(
 			cmd,
 			"--log-dir="+dockerlogDir,
-			"--logtostderr=false",
+			"--logtostderr=true",
```

Comment:
Eh, you don't want to log to files? Note that the previous code was logging to files and to stderr (due to the `--alsologtostderr=INFO` flag).